### DISCUSSION TITLE
Autocomplete: Show overlay panel even on empty input.

### DISCUSSION THREAD

**Discussion by Et7f3** [primefaces/community#2028] [PrimeReact]
I checked if it is not a bug in my code and it seems that is the behavior wanted: https://github.com/primefaces/primereact/blob/38faf1e80b897142468aed28996405204da27b7b/components/lib/autocomplete/AutoComplete.js#L73-L75

My users expected like google that overlay panel open even on empty input. I have default suggestions on empty input so it would be cool if a props allow this to be configured.

**Comment by melloware** [comment]
Isn't this the `emptyMessage` and `showEmptyMessage` properties?

**Reply by Et7f3** [reply]
In my case I want that is input is being focused it show the list of availables items. my function search doesn't return an empty list for empty string (because user just focused) so I don't think they would be useful.
